starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -3,13 +3,64 @@
 """
 
 from __future__ import annotations
+from itertools import islice
+from queue import Empty
+from typing import Union, List
+import collections
+from starknet_devnet.blueprints.rpc.structures.types import (
+    BlockId,
+    Felt,
+    Address,
+)
+from starknet_devnet.blueprints.rpc.structures.responses import (
+    RpcEventsResult,
+)
 
-from typing import Union
-
-from starknet_devnet.blueprints.rpc.structures.payloads import Felt, Address
 from starknet_devnet.state import state
 
 
+def is_empty(value):
+    """
+    Return true if empty.
+    """
+    return not value and isinstance(value, collections.Iterable)
+
+
+def is_not_empty(value):
+    """
+    Return true if not empty.
+    """
+    return not is_empty(value)
+
+
+def filter_address(address, event):
```

Comment:
These `filter_` functions have odd names. I would expect them to actually filter something. Whereas by looking into their code, it seems they are just indicators if something should be filtered or not.

Reply:
fixed